src/constants/categorical_palettes.ts

```diff
@@ -0,0 +1,235 @@
+import { Color } from "three";
+import { RawColorData } from "./color_ramps";
+
+export type PaletteData = RawColorData & {
+  colors: Color[];
+};
+
+const rawPaletteData: RawColorData[] = [
+  {
+    key: "matplotlib-accent",
+    name: "Accent",
+    colorStops: [
+      "#F25FA4",
+      "#7598C5",
+      "#E8E895",
+      "#AC8FD2",
+      "#A8D9AA",
+      "#BD5D26",
+      "#BEAED2",
+      "#5D5D6A",
+      "#FFFFA4",
+      "#3A6CAC",
+      "#ED1B7D",
+      "#83C885",
+    ],
+  },
+  {
+    key: "matplotlib-tab10",
+    name: "Set 1",
```

Comment:
This may be a question for UI but it seems like these could have better names? I know you have the preview of the colors so it's not as critical

Reply:
The names are designed to be flexible since everything is keyed with its own constant name. I'll check with Lyndsay on this one.

Were you thinking that they would have more colloquial names (like "Tropical" or "Marsh") or something else?

Reply:
Gotcha!